### DISCUSSION TITLE
Preci-Dip Connector Has 10 Pins?

### DISCUSSION THREAD

**Discussion by entekadesign** [ThomasPrior/LenovoSmartClock2#7] [Q&A] [answered]
Apologies in advance, if the following contains ridiculous questions, but I am new to electronics. I am intrigued by rooting my Lenovo clock by following your method, and would like to build the optional base with Preci-Dip connector. Your link to the Mouser site shows an "8-position" connector with 10 pins: the Lenovo clock has 8 holes to allow access to the 8 pin contacts on the PCB inside. Doesn't that mean that the Mouser connector has two too many pins, and won't they interfere with a good fit (acknowledging that the pins are spring-loaded)?

**Comment by ThomasPrior** [comment]
Looks like the page listing has changed. The product ID is the same.

**Reply by ThomasPrior** [reply]
You may be able to remove the extra pins with a Dremel or similar

**Reply by entekadesign** [reply, marked as answer]
I suppose that adjustments would be necessary to the 3D printable base designs linked to on your page. I happen to have an original Lenovo recharger dock; I will probably just bypass the PCB inside the dock and wire the USB jack directly to the connector.
